.snyk

```diff
@@ -1,4 +1,10 @@
 # Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
-version: v1.22.0
-ignore: {}
+version: v1.22.1
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-JS-AXIOS-1579269:
+    - '*':
+        reason: None Given
```

Comment:
Minor suggestions, the FFC team team to enter 'no fix available' in this situation.